applications/openshift/master/file_permissions_kube_apiserver/rule.yml

```
@@ -5,7 +5,7 @@ prodtype: ocp4
 title: 'Verify Permissions on the Kube API Server Pod Specification File'
 
 description: |-
-    {{{ describe_file_permissions(file="/etc/kubernetes/static-pod-resources/kube-apiserver-pod.yaml", perms="0644") }}}
+    {{{ describe_file_permissions(file="/etc/kubernetes/static-pod-resources-*/kube-apiserver-pod.yaml", perms="0644") }}}
```

The description is not matching with the actual filepath
```suggestion
    {{{ describe_file_permissions(file="/etc/kubernetes/static-pod-resources/kube-apiserver-pod-.*/kube-apiserver-pod.yaml", perms="0644") }}}
```